docs/coding-conventions/class-generation.md

```
@@ -6,18 +6,18 @@ Conventions for designing classes and methods. Most of what follows is reinforce
 
 ## Subsystem Layout
 
-Every mechanism in `frc.robot` is three pieces in one folder:
+Every mechanism in `frc.robot.subsystems` is one file holding the subsystem plus its config inner class:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)